vhdbuilder/packer/linux-packer-plugin.pkr.hcl

```diff
@@ -0,0 +1,8 @@
+packer {
+  required_plugins {
+    azure = {
+      version = ">= 2.1.7"
```

Comment:
does the current version we use in the dev container not support shallow replication?

Reply:
The azure-arm packer plugin did not have the shallow replication feature until v2.0.1 released in October 2023. Our packer version with the go dev container was released in August 2023, so it doesn't have the shallow replication functionality. New packer versions don't have the plugins bundled ahead of time, so I had to add this. I made a new file instead of changing the old one because windows is using that plugin.